### ISSUE TITLE
Allow block tags (#minecraft:wool) to be able to be added to xray

### ISSUE THREAD

**Issue by Synthestra** [ImpactDevelopment/ImpactClient#1053]
### What is your suggestion?

Allow block tags (#minecraft:wool) to be able to be added to xray

### Why is it useful?

Since 1.13 has split everything into its own ids, it's easier than typing .xray add red_wool, .xray add black_wool, .xray add white_wool though all 16 blocks. Using built in block tags would be great.

**Comment by LeafHacker** [comment]
This is a good idea, although it might be hard to add to the GUI... Maybe make the search box a bit smarter and also filter by tag, and allow manual entry via search & `Enter`?

**Comment by ZeroMemes** [comment]
Yeah we'll have to upgrade our "BlockManager" to support tags as well.

**Comment by ZeroMemes** [comment]
Not going to be doing this for 4.7

**Comment by Cerbiac** [comment]
You know 4.7 is close when they start adverting for 4.8.
